integration-commons/src/test/java/org/symphonyoss/integration/logging/LogMessageSourceTest.java

```diff
@@ -0,0 +1,49 @@
+package org.symphonyoss.integration.logging;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+
+/**
+ * Unit test for {@link LogMessageSource}
+
+ * Created by evandro on 31/01/17.
+ */
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@EnableConfigurationProperties
+@ContextConfiguration(classes = {LogMessageSource.class})
+public class LogMessageSourceTest {
+
+  @Autowired
+  LogMessageSource logMessage;
+
+  @Test
+  public void testCommonsMessage() {
+
+    assertEquals("Unable to load log messages from log-messages.properties files. "
```

Comment:
This is testing the text in the resource bundle in the test directory (which right now is a copy of the one in src directory.  But if someone changes that one, this test will still pass.  Is that expected?